docker/Dockerfile.prod

```diff
@@ -18,7 +18,7 @@ ENV PG_MAJOR 9.5
 ENV PG_VERSION 9.5.4-1.pgdg80+1
 RUN echo 'deb http://apt.postgresql.org/pub/repos/apt/ jessie-pgdg main' $PG_MAJOR > /etc/apt/sources.list.d/pgdg.list
 RUN apt-get update \
-    && apt-get install -y --no-install-recommends postgresql-client-$PG_MAJOR=$PG_VERSION \
+    && apt-get install -y --no-install-recommends postgresql-client-$PG_MAJOR \
```

Comment:
I removed $PG_VERSION because when building using develop.sh, apt-get gave errors saying no such version. I'll fix this by updating with the current PG_VERSION, if you wish, or I'll get this out of the commit.